media/js/base/stat-counter.js

```diff
@@ -0,0 +1,11 @@
+var STATCOUNTER_ID = document.getElementsByTagName('html')[0].getAttribute('data-statcounter-id');
```

Comment:
The analytic tool relies upon the sc_security and sc_project in the global scope. They are referenced from within `counter.js` (loaded by StatCounter) as `window.sc_project` and `window.sc_security`